native-sql-engine/core/src/main/scala/com/intel/oap/expression/ColumnarUnaryOperator.scala

```diff
@@ -1078,6 +1111,8 @@ object ColumnarUnaryOperator {
       new ColumnarHex(child)
     case _: Bin =>
       new ColumnarBin(child)
+    case _: Md5 =>
+      new ColumnarMd5(child)
     case other =>
```

Comment:
maybe also add Sha/Sha1/Sha2?looks the API is same

Reply:
Just supported them in latest commits. BTW, calling `sha` will actually invoke `sha1`.